src/types.ts

```diff
@@ -1,24 +1,176 @@
 export interface Coffeeshop {
-    name: string,
-    address: string
-    imageUrl: string,
-    drivingTime: DrivingTime
-  }
-  
-  export interface Restaurant {
-    name: string, 
-    address: string,
-    imageUrl: string,
-    rating: number,
-    currentlyOpen: boolean
-  }
+  name: string
+  address: string
+  imageUrl: string
+  drivingTime: DrivingTime
+}
+
+export interface Restaurant {
+  name: string
+  address: string
+  imageUrl: string
+  rating: number
+  currentlyOpen: boolean
+}
 
 export interface Coordinates {
-    lat: string
-    long: string
-  }
+  lat: string
+  long: string
+}
 
 export interface DrivingTime {
-    minutes: number
-    seconds: number
-}
\ No newline at end of file
+  minutes: number
+  seconds: number
+}
+
+export enum Genre {
+  American = "American",
+  Asian = "Asian",
+  Boba = "Boba",
+  Brunch = "Brunch",
+  Mediterranean = "Mediterranean",
+  Mexican = "Mexican",
+  Italian = "Italian",
+}
+
+export interface RestaurantDTO {
+  id: string
+  genre: Genre
+}
+
+export const restaurants = {
```

Comment:
this was motivated by the fact that genre just looked like a way to group the data. if there are multiples and they share a value, it just felt natural to me to do it this way. this is the main reason i went the direction I did, so if this does not sit right with u that makes sense bc this stuff will just have varying opinions

Reply:
I think this makes a lot of sense especially in the context of how i'm currently using this structure. Next step is adding restaurants to this list but I think that can be easily done since I don't have a sort order of these restaurants (which may come up later but that's a later problem)